deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml

```diff
@@ -0,0 +1,1150 @@
+{{- if .Values.features.autoscaler.enabled -}}
+# add autoscaler BOSH DNS alias
+- type: replace
+  path: /addons/name=bosh-dns-aliases/jobs/name=bosh-dns-aliases/properties/aliases/-
+  value:
+    domain: autoscalerpostgres.service.cf.internal
+    targets:
+    - deployment: cf
```

Comment:
The value here and below should be `{{ .Values.deployment_name | quote }}`.

Reply:
All the `deployment` value has been changed to `{{ .Values.deployment_name | quote }}`